package.json

```diff
@@ -9,6 +9,7 @@
     "react-dom": "^17.0.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.1",
+    "react-slider": "^1.3.1",
```

Comment:
Curiosity 😄 : Is there any reason why you don't use **Reach UI** [Slider component](https://reach.tech/slider/)?

Reply:
Hi @alxmcr, we have followed Andrew’s suggestion and are using an accessible slider suggested by Andrew last week.

Reply:
OK 👍 . I asked this question because Reach UI (an NPM dependency in our project) has a slider component, and it's possible that it wasn't necessary to install another dependency (increase the size in our app). I thought you tried to use it and you had some problems. But, as it was a suggestion. I think it's a better solution 👍 .

Reply:
I like what you did with the slider but I also like the ReachUI elements, ultimately it seems either one would work since they're both accessibility tested.